submissions/unabyband/popup-task/main.css

```diff
@@ -0,0 +1,302 @@
+html, body {
+    font-family: Arial, Helvetica, sans-serif;
+    width: 100%;
+    height: 100%;
+    margin: 0;
```

Comment:
for resetting default styles, you can use 
*{margin: 0; padding: 0;}
 It is a common practice
https://www.webfx.com/blog/web-design/css-tip-1-resetting-your-styles-with-css-reset/

Reply:
Thanks for advice, done!